### ISSUE TITLE
Text Similarity Notebook or Example

### ISSUE THREAD

**Issue by laxmimerit** [amaiya/ktrain#288]
Is there any text similarity notebook or example present in the repo?

**Comment by amaiya** [comment]
There are multiple approaches to computing semantic similarity.  One approach is to use LDA, which is covered in [this notebook](https://nbviewer.jupyter.org/github/amaiya/ktrain/blob/master/tutorials/tutorial-05-learning_from_unlabeled_text_data.ipynb). This, however, requires you to train an LDA model on a corpus.

If you're interested in computing text similarity using pretrained **transformers** models - either using a pre-fined-tuned model (e.g., MNLI)) or a model you fine-tuned yourself, you can just take the vector representation of the text and compute the distance/similarity using some metric with something like this:
```python
from scipy import spatial

dist_1 = spatial.distance.cosine(vectors[0], vectors[1])
dist_2 = spatial.distance.cosine(vectors[0], vectors[2])
print('dist_1: {0}, dist_2: {1}'.format(dist_1, dist_2))
```

See [this FAQ](https://github.com/amaiya/ktrain/blob/master/FAQ.md#how-do-i-obtain-the-word-or-sentence-embeddings-after-fine-tuning-a-transformer-based-text-classifier) for how to obtain a vector from text using your own fine-tuned model.  Or, you can use packages specifically for this like [sentence-transformers](https://github.com/UKPLab/sentence-transformers) or [sentvec](https://github.com/pdrm83/sent2vec).

**Comment by laxmimerit** [comment]
Thanks a lot for a quick response. I really appreciate it. Much helpful. Thanks.
